src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/jpa/entity/DefendantEntityTest.kt

```diff
@@ -17,6 +17,7 @@ internal class DefendantEntityTest {
       title = "Mr",
       familyName = "familyName",
       givenName = "givenName",
+      middleNames = listOf("middleName1", "middleName2"),
       defendantId = "defendantId",
```

Comment:
Not sure we really need this test - the next test src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/jpa/repository/DefendantRepositoryIntTest.kt seems to provide coverage of this entity.